src/core/Toast/Toast.tsx

```diff
@@ -132,6 +140,24 @@ export const Toast = (props: ToastProps) => {
     }
   };
 
+  const onAnimationRef = (ref: HTMLDivElement) => {
+    if (ref && animateOutTo) {
+      const { x, y } = ref.getBoundingClientRect();
+      setAnimateOutX(animateOutTo.getBoundingClientRect().x);
+      setAnimateOutY(animateOutTo.getBoundingClientRect().y);
+      setAnimationX(x);
+      setAnimationY(y);
+    }
+  };
+
+  const style = { height, marginBottom: visible ? '0' : -height };
+
+  const _transition = {
```

Comment:
I would say to have everything under `style` variable because current namings are a bit misleading. Or you can even have everything under `style` prop like it was before.